packages/newspringchurchapp/src/prayer/PrayerList/index.js

```diff
@@ -114,7 +114,7 @@ class PrayerList extends PureComponent {
     const { queryTag, label, variables } = this.calculateQuery();
 
     return (
-      <ModalView onClose={() => this.props.navigation.pop()}>
+      <ModalView onClose={() => this.props.navigation.navigate('Prayer')}>
```

Comment:
```suggestion
      <ModalView onClose={() => this.props.navigation.popToTop()}>
```

What do you think about this?